x-pack/plugins/apm/public/components/app/correlations/ml_failure_correlations.tsx

```diff
@@ -0,0 +1,410 @@
+              { defaultMessage: 'Filter out value' }
+            ),
+            icon: 'minusInCircle',
+            type: 'icon',
+            onClick: (term: FailureCorrelationValue) => {
+              push(history, {
+                query: {
+                  kuery: `not ${term.fieldName}:"${encodeURIComponent(
+                    term.fieldValue
+                  )}"`,
+                },
+              });
+              handleOnFilter();
+              trackApmEvent({ metric: 'correlations_term_exclude_filter' });
+            },
+          },
+        ],
+        name: i18n.translate(
+          'xpack.apm.correlations.correlationsTable.actionsLabel',
+          { defaultMessage: 'Filter' }
+        ),
+        render: (_: any, term: FailureCorrelationValue) => {
+          return (
+            <>
+              <EuiLink
+                href={createHref(history, {
+                  query: {
+                    kuery: `${term.fieldName}:"${encodeURIComponent(
+                      term.fieldValue
+                    )}"`,
+                  },
+                })}
+              >
+                <EuiIcon type="magnifyWithPlus" />
+              </EuiLink>
+              &nbsp;/&nbsp;
+              <EuiLink
+                href={createHref(history, {
+                  query: {
+                    kuery: `not ${term.fieldName}:"${encodeURIComponent(
+                      term.fieldValue
+                    )}"`,
+                  },
+                })}
+              >
+                <EuiIcon type="magnifyWithMinus" />
+              </EuiLink>
+            </>
+          );
+        },
+      },
+    ],
+    [handleOnFilter, history, trackApmEvent]
+  );
+
+  useEffect(() => {
+    if (isErrorMessage(error)) {
+      notifications.toasts.addDanger({
+        title: i18n.translate('xpack.apm.correlations.failure.errorTitle', {
+          defaultMessage:
+            'An error occurred performing correlations on failed transactions',
+        }),
+        text: error.toString(),
+      });
+    }
+  }, [error, notifications.toasts]);
+  return (
+    <>
+      <EuiFlexGroup>
+        <EuiFlexItem grow={false}>
+          {!isRunning && (
+            <EuiButton size="s" onClick={startFetch}>
+              <FormattedMessage
+                id="xpack.apm.correlations.failure.refreshButtonTitle"
+                defaultMessage="Refresh"
+              />
+            </EuiButton>
+          )}
+          {isRunning && (
+            <EuiButton size="s" onClick={cancelFetch}>
+              <FormattedMessage
+                id="xpack.apm.correlations.failure.cancelButtonTitle"
+                defaultMessage="Cancel"
+              />
+            </EuiButton>
+          )}
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiFlexGroup direction="column" gutterSize="none">
+            <EuiFlexItem data-test-subj="apmCorrelationsfailedTransactionsCorrelationsProgressTitle">
```

Comment:
`f` of the `failed` part should be uppercase: `apmCorrelationsfailedTransactionsCorrelationsProgressTitle`

Reply:
Thanks for catching that! Updated here https://github.com/elastic/kibana/pull/108441/commits/155443605ee491b2eb7c2bcb7b3e458eed9591e6